Changes to arch/powerpc/platforms/85xx/Makefile and Kconfig.

Keep these separate from the individual platform patches because all
platforms need to touch these two files.  By keeping these changes in
a separate patch allows the platform specific patchsets to sink/float
without conflicts in these two files.

diff --git a/arch/powerpc/platforms/85xx/Kconfig b/arch/powerpc/platforms/85xx/Kconfig
index b9a2559..2c1aa86 100644
--- a/arch/powerpc/platforms/85xx/Kconfig
+++ b/arch/powerpc/platforms/85xx/Kconfig
@@ -227,6 +227,99 @@ config P4080_DS
 	help
 	  This option enables support for the P4080 DS board
 
+config ACCTON_AS4600_54T
+       bool "Accton Technology Corporation AS4600_54T"
+       select DEFAULT_UIMAGE
+       help
+	 This option enables support for the ACCTON AS4600_54T networking platform
+
+config ACCTON_AS5610_52X
+       bool "Accton Technology Corporation AS5610_52X"
+       select DEFAULT_UIMAGE
+       help
+	 This option enables support for the ACCTON AS5610_52X networking platform
+
+config ACCTON_AS6701_32X
+       bool "Accton Technology Corporation AS6701-32X"
+       select DEFAULT_UIMAGE
+       help
+	 This option enables support for the Accton AS6701-32X networking platform
+
+config ACCTON_5652
+       bool "Accton Technology Corporation ES5652BT1"
+       select DEFAULT_UIMAGE
+       help
+	 This option enables support for the ACCTON ES5652BT1 networking platform
+
+config BCM98548XMC
+       bool "Broadcom BCM98548XMC"
+       select DEFAULT_UIMAGE
+       help
+         This option enables support for the Broadcom BCM98548XMC board used on reference platforms
+
+config CUMULUS_P2020
+       bool "Cumulus Networks P2020"
+       select DEFAULT_UIMAGE
+       help
+         This option enables support for the Cumulus P2020 networking platform
+
+config DNI_6448
+       bool "Delta Networks 6448"
+       select DEFAULT_UIMAGE
+       help
+         This option enables support for the DNI-6448 networking platform
+
+config DNI_7448
+       bool "Delta Networks 7448"
+       select DEFAULT_UIMAGE
+       help
+         This option enables support for the DNI-7448 networking platform
+
+config DNI_C7448N
+       bool "Delta Networks C7448N"
+       select DEFAULT_UIMAGE
+       help
+         This option enables support for the DNI-C7448N networking platform
+
+config QUANTA_LB8
+	bool "Quanta Computer LB8"
+	select DEFAULT_UIMAGE
+	select SWIOTLB
+	help
+          Enable support for the Quanta Computer LB8 48x10GE networking platform
+
+config QUANTA_LB9
+	bool "Quanta Computer LB9"
+	select DEFAULT_UIMAGE
+	help
+	  Enable support for the Quanta Computer LB9 48x1G networking platform
+
+config QUANTA_LY2_LY2R
+	bool "Quanta Computer LY2 and LY2R"
+	select DEFAULT_UIMAGE
+	help
+	  Enable support for the Quanta Computer LY2 based networking platforms
+
+config QUANTA_LY2
+	bool "Quanta Computer LY2"
+	select DEFAULT_UIMAGE
+	select QUANTA_LY2_LY2R
+	help
+	  Enable support for the Quanta Computer LY2 48x10GE networking platform
+
+config QUANTA_LY2R
+	bool "Quanta Computer LY2R"
+	select DEFAULT_UIMAGE
+	select QUANTA_LY2_LY2R
+	help
+	  Enable support for the Quanta Computer LY2R 48x10GE networking platform
+
+config QUANTA_LY6_P2020
+	bool "Quanta Computer LY6, P2020"
+	select DEFAULT_UIMAGE
+	help
+	  Enable support for the Quanta Computer LY6 32x40G networking platform
+
 endif # PPC32
 
 config P5020_DS
diff --git a/arch/powerpc/platforms/85xx/Makefile b/arch/powerpc/platforms/85xx/Makefile
index bc5acb9..d04e66c 100644
--- a/arch/powerpc/platforms/85xx/Makefile
+++ b/arch/powerpc/platforms/85xx/Makefile
@@ -25,3 +25,16 @@ obj-$(CONFIG_SBC8548)     += sbc8548.o
 obj-$(CONFIG_SOCRATES)    += socrates.o socrates_fpga_pic.o
 obj-$(CONFIG_KSI8560)	  += ksi8560.o
 obj-$(CONFIG_XES_MPC85xx) += xes_mpc85xx.o
+obj-$(CONFIG_ACCTON_AS4600_54T)  += accton_as4600_54t.o
+obj-$(CONFIG_ACCTON_AS5610_52X)  += accton_as5610_52x.o
+obj-$(CONFIG_ACCTON_AS6701_32X) += accton_as6701_32x.o
+obj-$(CONFIG_ACCTON_5652) += accton_5652.o
+obj-$(CONFIG_BCM98548XMC) += bcm98548xmc.o
+obj-$(CONFIG_CUMULUS_P2020) += cumulus_p2020.o
+obj-$(CONFIG_DNI_6448)    += dni_6448.o
+obj-$(CONFIG_DNI_7448)    += dni_7448.o
+obj-$(CONFIG_DNI_C7448N)  += dni_c7448n.o
+obj-$(CONFIG_QUANTA_LB8)  += quanta_lb8.o
+obj-$(CONFIG_QUANTA_LB9)  += quanta_lb9.o
+obj-$(CONFIG_QUANTA_LY2_LY2R)  += quanta_ly2_ly2r.o
+obj-$(CONFIG_QUANTA_LY6_P2020)  += quanta_ly6_p2020.o
